addon/modifiers/did-resize.js

```diff
@@ -6,23 +6,41 @@ export default class DidResizeModifier extends Modifier {
   options = {};
 
   // Private API
-  observer = null;
+  _didInstall = false;
+  static observer = null;
+  static handlers = [];
+
+  addHandler() {
+    DidResizeModifier.handlers.push({
+      element: this.element,
+      handler: this.handler,
+    });
+  }
+
+  removeHandler() {
+    DidResizeModifier.handlers.splice(
+      DidResizeModifier.handlers.findIndex((h) => h.element === this.element),
```

Comment:
Since we're using `this.element` as the lookup key, would it makes sense to use a WeakMap here instead where the DOM element itself is the key and the handler is the value?